set.mm

```diff
@@ -79592,21 +79592,544 @@ currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
+       11-Sep-2023.) $)
+    fprlem1 $p |- ( ( ( R Fr A /\ R Po A /\ R Se A ) /\
+      ( g e. B /\ h e. B ) ) -> ( ( x g u /\ x h v ) -> u = v ) ) $=
+      ( va cv wbr wa cres wceq cdm cin wfr wpo wse w3a wcel weq vex breldm elin
+      biimpri syl2an brresi anbi12i an4 bitri syl21anbrc wfn cfv cpred wral wss
+      id inss2 frrlem3 sstrid adantl simpl1 frss sylc simpl2 poss sess2 frrlem4
+      co simpl3 incom reseq2i fneq12 mp2an fveq1i predeq2 ax-mp reseq12i oveq2i
+      eqeq12i raleqbii sylibr ancoms fpr3g syl311anc breqd biimprd wfun frrlem2
+      wb wi ad2antrl funres wrel wal dffun2 2sp sps simplbiim 3syl sylan2d syl5
+      ) APZDPZIPZQZXJCPZJPZQZRZXJXKXLXLUAZXOUAZUBZSZQZXJXNXOXTSZQZRZEGUCZEGUDZE
+      GUEZUFZXLFUGZXOFUGZRZRZDCUHZXQXJXTUGZYOXQYEXMXJXRUGZXJXSUGZYOXPXJXKXLAUIZ
+      DUIZUJXJXNXOYRCUIZUJYOYPYQRXJXRXSUKULUMZUUAXQVDYEYOXMRZYOXPRZRYOYORXQRYBU
+      UBYDUUCXTXJXKXLYSUNXTXJXNXOYTUNUOYOXMYOXPUPUQURYMYDXJXNYAQZYBYNYMUUDYDYMY
+      AYCXJXNYMXTGUCZXTGUDZXTGUEZYAXTUSOPZYAUTUUHYAXTGUUHVAZSLVPTOXTVBRZYCXTUSZ
+      UUHYCUTZUUHYCUUISZLVPZTZOXTVBZRZYAYCTYMXTEVCZYFUUEYLUURYIYKUURYJYKXTXSEXR
+      XSVEABEFGHJLMVFVGVHVHZYFYGYHYLVIXTEGVJVKYMUURYGUUFUUSYFYGYHYLVLXTEGVMVKYM
+      UURYHUUGUUSYFYGYHYLVQXTEGVNVKYLUUJYIABEFGHIJLOMVOVHYLUUQYIYKYJUUQYKYJRXOX
+      SXRUBZSZUUTUSZUUHUVAUTZUUHUVAUUTGUUHVAZSZLVPZTZOUUTVBZRUUQABEFGHJILOMVOUU
+      KUVBUUPUVHYCUVATXTUUTTZUUKUVBWQXTUUTXOXRXSVRZVSZUVJXTUUTYCUVAVTWAUUOUVGOX
+      TUUTUVJUULUVCUUNUVFUUHYCUVAUVKWBUUMUVEUUHLYCUVAUUIUVDUVKUVIUUIUVDTUVJXTUU
+      TGUUHWCWDWEWFWGWHUOWIWJVHOXTGYAYCLWKWLWMWNYMXLWOZYAWOZYBUUDRYNWRZYJUVLYIY
+      KABEFGHILMWPWSXTXLWTUVMYAXAUVNCXBDXBZAXBUVNADCYAXCUVOUVNAUVNDCXDXEXFXGXHX
+      I $.
+  $}
+
+  ${
+    $d R w x z $.  $d A w x z $.
+    $( Lemma for well-founded recursion with a partial order.  Establish a
+       subset relationship.  (Contributed by Scott Fenton, 11-Sep-2023.) $)
+    fprlem2 $p |- ( ( ( R Fr A /\ R Po A /\ R Se A ) /\ z e. A ) ->
+       A. w e. Pred ( R , A , z ) Pred ( R , A , w ) C_ Pred ( R , A , z ) ) $=
+      ( vx wfr w3a cv wcel wa cpred wbr wb cvv vex elpred elv simprl adantr jca
+      wpo wse wss simpll2 simpllr 3jca simprr simplrr potr sylc 3imtr4g sylan2b
+      ex ssrdv ralrimiva ) CDFZCDUAZCDUBZGZAHZCIZJZCDBHZKZCDUTKZUCZBVEVCVEIZVBV
+      CCIZVCUTDLZJZVFVGVJMACNDUTVCBOPQVBVJJZEVDVEVKEHZCIZVLVCDLZJZVMVLUTDLZJZVL
+      VDIZVLVEIZVKVOVQVKVOJZVMVPVKVMVNRZVTUQVMVHVAGZJVNVIJVPVTUQWBVKUQVOUPUQURV
+      AVJUDSVTVMVHVAWAVKVHVOVBVHVIRSUSVAVJVOUEUFTVTVNVIVKVMVNUGVBVHVIVOUHTCVLVC
+      UTDUIUJTUMVRVOMBCNDVCVLEOZPQVSVQMACNDUTVLWCPQUKUNULUO $.
+  $}
+
+  ${
+    $d F x y z u v a b c f g h $.  $d R x y z u v a b c f g h $.
+    $d A x y z u v a b c f g h $.  $d G x y z u v a b c f g h $.
+    fprr.1 $e |- F = frecs ( R , A , G ) $.
+    $( Law of well-founded recursion over a partial order, part one.  Establish
+       the functionhood and domain of the recursive function generator.  Note
```

Comment:
Here and above/below, I'd write "functionality".

Reply:
Done